website/docs/r/vapp_org_network.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_org_network"
+sidebar_current: "docs-vcd-resource-vapp-org-network"
+description: |-
+  Provides a vCloud Director vApp Org Network. This can be used to create and delete internal Org networks for vApps to connect.
+---
+
+# vcd\_vapp\_org\_network
+
+ Provides a vCloud Director vApp Org Network. This can be used to create and delete internal Org networks for vApps to connect.
+
```

Comment:
We need to add a note - an overview - here (and in other vApp network doc page) with a list of available vApp network types and with which resource and field you can achieve that. Right now it's hard to make sense of why we have two resources. Something like:

- Isolated - vcd_vapp_network
- Attached (?!) to Org net (?) - vcd_vapp_network.org_network_name
- Routed to Org (?) - vcd_vapp_org_network
- ...

DISCLAIMER: this is just an example, adjust to make true sense

Reply:
Changed description according idea from Dainius.

Reply:
At least for me this is clear now.